app/furniture/marketplace/order/mailer.rb

```diff
@@ -0,0 +1,15 @@
+class Marketplace
+  class Order
+    class Mailer < ApplicationMailer
+      attr_reader :order
+      helper_method :order
```

Comment:
I don't understand the purpose of these two lines. Is this just so that the template can refer to `order` instead of `@order`, or am I missing something else?

Reply:
Yes, that's exactly it. I am allergic to instance variables in my templates to the point where I will do absurd things to avoid them.